src/libcollections/slice.rs

```diff
@@ -188,15 +188,36 @@ pub trait SliceExt {
     /// indices from `[mid, len)` (excluding the index `len` itself).
     ///
     /// Panics if `mid > len`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let v = [10i32, 40, 30, 20, 50];
```

Comment:
I think in all of these examples you may be able to leave out the `i32` suffix entirely